lib/server/index.js

```diff
@@ -11,6 +11,14 @@ import passport from 'passport';
 import path from 'path';
 import safeHtmlString from 'helpers/safe-html-string';
 import session from 'express-session';
+import {
+  favicon,
+  title,
+  description,
+  keywords,
+  copyright,
+  robots
+} from 'helpers/const/setting-keys';
```

Comment:
Place `setting-keys` in statics folder https://github.com/relax/relax/tree/master/lib/shared/statics and rename to `settings-keys`